File: java/src/main/java/com/oscgp/dbcompare/common/Utility.java
/*
 * OSCG-Partners
 * 2022 All rights reserved 
 */
package com.oscgp.dbcompare.common;

import java.sql.Timestamp;

/**
 *
 * @author Muhammad Asif Naeem
 */
public class Utility {
    static public Timestamp getCurrentTime() {
        Timestamp timestamp = new Timestamp(System.currentTimeMillis());
        return timestamp;
    }

    static public void printLog(String text, boolean verbose) {
        if(verbose) {
            System.out.println(getCurrentTime() + ": " + text);
        }
    }
}

File: csharp/Utility.cs
using System;
namespace OSCGP.DBCompare
{
    public class Utility
    {
        public Utility() {}
        static public string getCurrentTime()
        {
            string timestamp = DateTime.Now.ToString("yyyy-MM-dd HH:mm:ss.fff");
            return timestamp;
        }

        static public void printLog(String text, Boolean verbose)
        {
            if (verbose)
            {
                Console.WriteLine(getCurrentTime() + ": " + text);
            }
        }
    }
}
